go-controller/pkg/ovn/egressservices_test.go

```diff
@@ -939,7 +1295,19 @@ func (o *FakeOVN) InitAndRunEgressSVCController(tweak ...func(*DefaultNetworkCon
 	}()
 }
 
-func lrpForEgressSvcEndpoint(uuid, key, addr, nexthop string) *nbdb.LogicalRouterPolicy {
+// creates a logical router static route for egress service
+func staticRoute(uuid, key, addr, nexthop string) *nbdb.LogicalRouterStaticRoute {
```

Comment:
nit: egressSvcStaticRoute? (due to the fixed externalid)

Reply:
This is a private function only used in that test file. 
I wanted to avoid unnecessarily long names and I provided a doc for the function stating that it is creating the route for an egress service.

Reply:
yep but it's all under the ovn package, so I assume it would be clearer to have a named difference (anyways it's not too critical feel free to leave it)

Reply:
I am ok to change that.